survey_resource_booking/models/resource_booking.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+from odoo.tests.common import Form
+from odoo.tools.mail import email_split
+
+
+class ResourceBooking(models.Model):
+    _inherit = "resource.booking"
+
+    survey_user_input_id = fields.Many2one(
+        comodel_name="survey.user_input",
+        string="Survey user input",
+        tracking=True,
+        help="User responses to survey, as specified in the resource booking type.",
+    )
```

Comment:
I think this field should not be editable by the user, or at the very least, it should not be copied. What do you think?
![image](https://github.com/user-attachments/assets/3fc98e3b-a351-4f2b-b417-a290b0f33edf)

```suggestion
    survey_user_input_id = fields.Many2one(
        comodel_name="survey.user_input",
        string="Survey user input",
        tracking=True,
        copy=False,
        readonly=True,
        help="User responses to survey, as specified in the resource booking type.",
    )
```

Reply:
`copy=false` added.

Although I understand what you indicate that this field is readonly in the view, perhaps in some cases it may be interesting to change it, therefore, I think it is better to leave it as it is in case someone is using this possibility.